frontend/pages/software/ManageSoftwarePage/SoftwareTableConfig.tsx

```diff
@@ -172,25 +193,12 @@ const generateTableHeaders = (isPremiumTier?: boolean): Column[] => {
       disableSortBy: true,
       accessor: "name",
       Cell: (cellProps: IStringCellProps): JSX.Element => {
-        const { name, bundle_identifier } = cellProps.row.original;
-        if (bundle_identifier) {
-          return (
-            <span className="name-container">
-              <TooltipWrapper
-                tipContent={`
-                  <span>
-                    <b>Bundle identifier: </b>
-                    <br />
-                    ${bundle_identifier}
-                  </span>
-                `}
-              >
-                {name}
-              </TooltipWrapper>
-            </span>
-          );
-        }
-        return <TextCell value={name} />;
+        const { id, name, bundle_identifier: bundle } = cellProps.row.original;
+        return (
+          <Link to={`${PATHS.SOFTWARE_DETAILS(id.toString())}`}>
+            {bundle ? withBundleTooltip(name, bundle) : name}
+          </Link>
+        );
```

Comment:
@lukeheath yes, I think so. It is included in the `GET /software` response for me (although I did have to search for a macOS application since one doesn't appear on the first page of the results).